docs/UserGuide.md

```diff
@@ -2,15 +2,25 @@
 layout: page
 title: User Guide
 ---
-
-Vax@NUS is a **one stop management app to efficiently track and schedule COVID-19 vaccinations for NUS students.** It is a desktop app **optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Vax@NUS can get your appointment management tasks done faster than traditional GUI apps.
-
 * Table of Contents
 {:toc}
 
 :information_source: This icon indicates helpful notes 
 
 --------------------------------------------------------------------------------------------------------------------
+## Introduction
+
+Vax@NUS was conceived at the start of Singapore's COVID-19 vaccination campaign. Our team sought to streamline the 
+vaccine rollout for the University Health Centre(UHC), responsible for administering vaccines to NUS students, when they become eligible to receive the vaccine. 
+Feedback gathered from staff revealed that existing systems were be slow, and difficult to use, potentially impacting the speed of the rollout. 
```

Comment:
Should it be "the existing system" rather than "existing systems"? and "was/were slow" instead of "were be slow," (i.e. remove "be" and the comma)?

Reply:
Ok done